src/libraries/System.Private.CoreLib/src/System/Int16.cs

```diff
@@ -631,24 +631,11 @@ public static short Log2(short value)
         /// <inheritdoc cref="INumber{TSelf}.CopySign(TSelf, TSelf)" />
         public static short CopySign(short value, short sign)
         {
-            short absValue = value;
-
-            if (absValue < 0)
-            {
-                absValue = (short)(-absValue);
-            }
-
-            if (sign >= 0)
+            if (short.MinValue != value || 0 > sign)
```

Comment:
We do not use the style that put constant on the left.

Reply:
Thank you for your review. I will change it to the existing style. Do you know any tool that can do this?

Reply:
I'm not aware of such rule in built-in analysis tool.